stats/zap_services.py

```diff
@@ -93,7 +93,7 @@ def convert_file(source, dest) :
                     daily += int(count.strip())
                     dday = date
                     ver = 'Daily'
-                elif ver == '2.12.0' or ver == '2.13.0':  # Will need to change for new releases
+                elif ver == '2.12.0' or ver == '2.13.0' or ver == '2.14.0':  # Will need to change for new releases
```

Comment:
Do we plan for this to be an ever increasing list?

Reply:
Agreed, not a blocker.

I was more wondering if it should be a window of x-3 releases or something. Or if moving to a list.contains kinda construct would be easier.